xbmc/addons/AddonManager.h

```diff
@@ -486,6 +486,14 @@ namespace ADDON
     bool GetCompatibleVersions(const std::string& addonId,
                                std::vector<std::shared_ptr<IAddon>>& compatibleVersions) const;
 
+    /*!
+     * \brief Return number of available updates formatted as string for infolabel use
```

Comment:
Don't want to be too nitpicking, but how could you know that this string is only used for GUI info labels? So, say this is an example use or (better) just leave this kind of usage assumptions.

BTW: It's "GUI info labels"

Reply:
What else would you use a string surrounded by `{}` for?

Reply:
Ignore me. Having a face palm moment.

Reply:
this was just an example, i'd simply cut `for infolabel use` off. agree this is the official doc and would imply it is only for gui infolabel usage which is not true... let's do it perfect :)